src/components/Item.tsx

```diff
@@ -57,6 +59,26 @@ const Item: React.FC<ItemProps> = ({ item, primaryColor }) => {
             : `${baseClass} ${selectedClass}`;
     }, [bgColor, bgHoverColor, classNames, isSelected, textHoverColor]);
 
+    const getLabel = useCallback(() => {
+        if (!noHighLigthLabel && searchInputValue) {
+            const start = item.label.toUpperCase().indexOf(searchInputValue.toUpperCase());
+            const end = start + searchInputValue.length;
+            return item.label.split("").map((label, idx) => {
+                if (idx >= start && idx < end) {
+                    return (
+                        <span key={idx} className={`font-bold text-${primaryColor}-500`}>
```

Comment:
Unfortunately, you can't use string interpolation like `text-${primaryColor}-500 ` as described in the Tailwind documentation https://tailwindcss.com/docs/content-configuration#dynamic-class-names

```
const getTextColor = useMemo(() => {
        if (COLORS.includes(primaryColor)) {
            return THEME_DATA.text[primaryColor as keyof typeof THEME_DATA.text];
        }
        return THEME_DATA.text[DEFAULT_THEME];
    }, [primaryColor]);
```

You can use this code to get the color of the text based on `primaryColor`. 
The code ([here](https://github.com/onesine/react-tailwindcss-select/blob/master/src/components/Item.tsx)) contains other examples.

Reply:
Thanks !! 👍